Simplified/NYPLReaderReadiumView.m

```diff
@@ -811,7 +811,7 @@ - (void)calculateBookLength
         [spineItemDict setObject:title forKey:@"tocElementTitle"];
       }
       else {
-        [spineItemDict setObject:NSLocalizedString(@"chapter", nil) forKey:@"tocElementTitle"];
+        [spineItemDict setObject:NSLocalizedString(@"ReaderViewControllerCurrentChapter", nil) forKey:@"tocElementTitle"];
```

Comment:
this key exists, right? i haven't checked , just making sure we have the actual localized string for this.